tonto/schemas.py

```diff
@@ -0,0 +1,20 @@
+from pydantic import BaseModel
+from typing import List, Optional
+from datetime import datetime, date
+
+
+# crear un cowboy
+class NuevoTonto (BaseModel):
+    cowboy_id:int
```

Comment:
En la solicitud de guardar tonto, el campo se llama `id`.

![image](https://github.com/The-Cowboys/tonto-apy/assets/25726643/b75a0903-82d3-4bb7-b5d7-b103bbdeac28)